transmart-copy/src/main/groovy/org/transmartproject/copy/Copy.groovy

```diff
@@ -228,8 +247,13 @@ class Copy implements AutoCloseable {
                     copy.uploadPedigree(directory, config)
                 }
                 if (!modes || 'study' in modes) {
-                    copy.deleteStudy(directory, false)
-                    copy.uploadStudy(directory, config)
+                    if (cl.hasOption('incremental')) {
+                        copy.deleteObservationsForPatientsInStudies(directory)
```

Comment:
I would expect this to be part of `uploadStudy`. The idea is that all data is read only once, single pass, and that the table classes are instantiated only once.